modules/logging-upgrading-clo.adoc

```diff
@@ -16,6 +16,34 @@ To update the {clo} to a new major release version, you must modify the update c
 
 .Procedure
 
+. Optional: When upgrading to the {clo} 5.9 or later, you must add the `v13` schema version to the `LokiStack` custom resource before the upgrade:
++
+[source,yaml]
+----
+spec:
+# ...
+  storage:
+    schemas:
+    - effectiveDate: "<yyyy>-<mm>-<dd>" # <1>
+      version: v12
+    - effectiveDate: "<yyyy>-<mm>-<dd>" # <2>
+      version: v13 # <3>
+# ...
+----
+<1> You can set today's date. Later, after the `effectiveDate` date of the new `v13` schema version, you can remove the `v12` schema version.
```

Comment:
It's not a "You can", it's a must as said in the upstream doc [0]:

  "for a new install, this must be a date in the past, use a recent date. Format is YYYY-MM-DD."

But, in the upgrade example, it's not relevant here as it's expected this entry exists and what's the important is to add the second containing the new schema

[0] https://grafana.com/docs/loki/latest/operations/storage/schema/#new-loki-installs

Reply:
I think that is an existing  item, isn't it? If yes. we have to remove 'You can set today's date'.

Reply:
Yes, it's the existent, then, that's not relevant and it could be removed.

Reply:
Thank you. Editing.